packages/SketchMorph2-Core.package/monticello.meta/initializers.st

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file seems to be empty, can it be deleted?

Reply:
I don't know. I don't think we've created it, but I assume that Squeak may expect it, especially since it's located in a folder called `monticello.meta`.